testapi.pm

```diff
@@ -1324,7 +1324,13 @@ Uses C<$testapi::password> if no string is given.
 sub type_password {
     my ($string, %args) = @_;
     $string //= $password;
-    type_string $string, secret => 1, max_interval => ($args{max_interval} // 100);
+    type_string($string,
```

Comment:
Can you make `type_password` an actual alias to `type_string` that passes all `%args` so the defaults etc. don't need to be re-defined here? The only difference would be that `secret => 1` is set.

Reply:
I was thinking about it, but there is already `max_interval` with different default for `type_password` and I wanted to follow same logic, although with same defaults for other arguments.

Reply:
Alright, that makes sense. I'd still suggest passing other arguments as-is since there's no requirement to have different defaults for everything.